libtransmission/magnet-metainfo.cc

```diff
@@ -180,8 +180,6 @@ std::string tr_magnet_metainfo::magnet() const
 
 bool tr_magnet_metainfo::parseMagnet(std::string_view magnet_link, tr_error** error)
 {
-    magnet_link = tr_strvStrip(magnet_link);
-
```

Comment:
This shouldn't've been removed -- this line lets magnet links that accidentally have leading or trailing whitespace still get parsed correctly

Reply:
Oope